config/.env.sample

```diff
@@ -17,33 +17,44 @@ LOG_LEVEL='INFO'
 # The .srt files must be stored within this folder.
 VIDEO_TO_USE = ''
 
+# Specify the type of question generation model to use.
+# Available options: 'BERTopic', 'LangChain'
+# Defaults to 'BERTopic'.
+GENERATION_MODEL = 'BERTopic'
```

Comment:
Looks like model values are case-sensitive and can be easily written without case. So something to think about it for future. I won't suggest a change but making it case-insensitive is better

Reply:
This is a good suggestion and should be quite easy to validate and apply. I will have this applied.